app/frontend/routes/index.js

```
@@ -0,0 +1,1955 @@
+            : _a.port)) ||
+        ""
+      );
+    }
+    is_object(value) {
+      return (
+        typeof value === "object" &&
+        Object.prototype.toString.call(value) === "[object Object]"
+      );
+    }
+    is_array(object) {
+      return object instanceof Array;
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)